updated auto clock update fixes and resolved conflict issue, PR against master

naseemshekh20 opened the pull request:
updated auto clock update fixes and resolved conflict issue
Added masthead changes (show time, component name) and backspase key issue (from main menu/after launch on back key app was crashing and user can't perform other operation)

piotr-serafin-epam reviewed (commented):
Current implementation adds all timer logic into root App component. For Time you should create similar component as it was in spark version of refapp:
https://github.com/LibertyGlobal/refapp/tree/83f9ba006f879677858fbd824496c1ec23034213/src/components/mastHead

naseemshekh20 commented:
> Current implementation adds all timer logic into root App component. For Time you should create similar component as it was in spark version of refapp:
> https://github.com/LibertyGlobal/refapp/tree/83f9ba006f879677858fbd824496c1ec23034213/src/components/mastHead

Auto clock update - Addressed review comment in PR- https://github.com/LibertyGlobal/refapp/pull/38

naseemshekh20 commented:
Resolved in PR: https://github.com/LibertyGlobal/refapp/pull/38
